packages/darklang/languageTools/semanticTokens.dark

```diff
@@ -167,10 +167,11 @@ module Darklang =
               [ makeToken symbolOpenQuote TokenType.Symbol ]
 
               // hello
-              (match contentsMaybe with
-               | Some(contentsRange, _contents) ->
-                 [ makeToken contentsRange TokenType.String ]
-               | None -> [])
+              [ makeToken range TokenType.String ]
+              // (match contentsMaybe with
+              //  | Some(contentsRange, _contents) ->
+              // [ makeToken contentsRange TokenType.String ]
+              //  | None -> [])
```

Comment:
I think you're right here, and that the commented stuff can be deleted (right?)

Reply:
Yes, I left it in case it's needed because I didn't understand how it was working. I'll delete it in the next commit